python-template/{{cookiecutter.placeholder_repo_name}}/docs/decisions/README.rst

```diff
@@ -0,0 +1,8 @@
+This directory is a historical record on the architectural decisions we make with this IDA as it evolves over time.
+
+It uses Architecture Decision Records, as described by Michael Nygard in `Documenting Architecture Decisions`_.
+
+For more information, see `OEP-19's section on ADRs`_.
```

Comment:
Mismatched link ID.

Reply:
I think this might be the only CI failure: https://github.com/openedx/edx-cookiecutters/runs/7322021018?check_suite_focus=true#step:6:2371